x-pack/plugins/canvas/public/plugin.tsx

```diff
@@ -126,6 +136,13 @@ export class CanvasPlugin
       plugins.home.featureCatalogue.register(featureCatalogueEntry);
     }
 
+    if (plugins.expressionRevealImage) {
```

Comment:
nit: just an idea, maybe we can modify API and wrap all into callback function? I like the following code more

    plugins.expressionRevealImage?.register(({ elements, views, renderers }) => {
      canvasApi.addElements(elements);
      canvasApi.addViewUIs(views);
      canvasApi.addRenderers(renderers);
      addSupportedRenderFunctions(renderers);
    });

Reply:
I enjoy this idea, but I'm not sure of the function's name.

Reply:
Done